### ISSUE TITLE
Warning regarding align.py

### ISSUE THREAD

**Issue by poke1024** [tomas-gajarsky/facetorch#74]
With python 3.10, the following warning pops up:

```
/home/xxx/anaconda3/envs/jupyter/lib/python3.10/site-packages/facetorch/analyzer/utilizer/align.py:223: UserWarning: Using torch.cross without specifying the dim arg is deprecated.
Please either pass the dim explicitly or simply use torch.linalg.cross.
The default value of dim will change to agree with that of linalg.cross in a future release. (Triggered internally at /opt/conda/conda-bld/pytorch_1712608935911/work/aten/src/ATen/native/Cross.cpp:62.)
  r3 = torch.cross(r1, r2)
```

**Comment by tomas-gajarsky** [comment]
Which PyTorch version have you used?

**Comment by tomas-gajarsky** [comment]
Thank you for reporting this! I’ve updated the code to use torch.linalg.cross instead of torch.cross to address this deprecation warning. The fix is included in version 0.5.1.

Let me know if you encounter any other issues!
